event-sourcing/event-publisher/linked-event-processor/src/test/java/uk/gov/justice/services/eventsourcing/linkedevent/ActiveStreamsProcessorIT.java

```diff
@@ -0,0 +1,163 @@
+import uk.gov.justice.services.eventsourcing.EventFetcherRepository;
+import uk.gov.justice.services.eventsourcing.linkedevent.helpers.EventFactory;
+import uk.gov.justice.services.eventsourcing.linkedevent.helpers.EventStoreInitializer;
+import uk.gov.justice.services.eventsourcing.linkedevent.helpers.TestEventInserter;
+import uk.gov.justice.services.eventsourcing.linkedevent.helpers.TestEventStreamInserter;
+import uk.gov.justice.services.eventsourcing.prepublish.LinkedEventFactory;
+import uk.gov.justice.services.eventsourcing.prepublish.MetadataEventNumberUpdater;
+import uk.gov.justice.services.eventsourcing.prepublish.PrePublishRepository;
+import uk.gov.justice.services.eventsourcing.repository.jdbc.AnsiSQLEventLogInsertionStrategy;
+import uk.gov.justice.services.eventsourcing.repository.jdbc.event.Event;
+import uk.gov.justice.services.eventsourcing.repository.jdbc.event.EventConverter;
+import uk.gov.justice.services.eventsourcing.repository.jdbc.event.EventJdbcRepositoryFactory;
+import uk.gov.justice.services.eventsourcing.repository.jdbc.event.LinkedEvent;
+import uk.gov.justice.services.eventsourcing.repository.jdbc.eventstream.EventStreamJdbcRepositoryFactory;
+import uk.gov.justice.services.jdbc.persistence.JdbcRepositoryHelper;
+import uk.gov.justice.services.test.utils.persistence.FrameworkTestDataSourceFactory;
+import uk.gov.justice.services.test.utils.persistence.TestJdbcDataSourceProvider;
+import uk.gov.justice.subscription.domain.eventsource.EventSourceDefinition;
+import uk.gov.justice.subscription.domain.eventsource.Location;
+import uk.gov.justice.subscription.registry.DefaultEventSourceDefinitionFactory;
+import uk.gov.justice.subscription.registry.EventSourceDefinitionRegistry;
+import uk.gov.justice.subscription.registry.SubscriptionDataSourceProvider;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.sql.Connection;
+import java.time.ZonedDateTime;
+import java.util.Optional;
+import java.util.UUID;
+
+import javax.sql.DataSource;
+
+import org.junit.Before;
+import org.junit.Test;
+
+public class ActiveStreamsProcessorIT {
+    private final DataSource eventStoreDataSource = new FrameworkTestDataSourceFactory().createEventStoreDataSource();
+    private final TestEventInserter testEventInserter = new TestEventInserter();
+    private final TestEventStreamInserter testEventStreamInserter = new TestEventStreamInserter();
+    private final EventFactory eventFactory = new EventFactory();
+
+    private EventStreamJdbcRepositoryFactory eventStreamJdbcRepositoryFactory = new EventStreamJdbcRepositoryFactory();
+    private EventJdbcRepositoryFactory eventJdbcRepositoryFactory = new EventJdbcRepositoryFactory();
+
+    ActiveStreamsProcessor activeStreamsProcessor;
+    final LinkedEventsProcessor linkedEventsProcessor = new LinkedEventsProcessor();
+    final LinkedEventProcessor linkedEventProcessor = new LinkedEventProcessor();
+
```

Comment:
Some fields are private and final, some are final and some are private. Should they not all be consistent?

Reply:
Agreed and fixed